docs/MatrixOne/Overview/mysql-compatibility.md

```diff
@@ -7,20 +7,23 @@ MatrixOne SQL syntax conforms with MySQL 8.0.23 version.
 | DDL  | CREATE DATABASE | A database with Chinese name is not supported. |
 |   |   | Names with Latins support limitedly.  |
 |   |   | `CHARSET`, `COLLATE`, `ENCRYPTION` can be used but don't work. |
-|   | CREATE TABLE | Temporary tables are not supported.  |
-|   |   | Partition tables are not supported.  |
+|   |  CREATE TABLE | Partition tables are not supported.  |
 |   |   | Create table .. as clause is not supported now. |
 |   |   | All column level constraints are not supported now. |
 |   |   | Composite Primary Key is not supported yet. |
 |   |   | KEY(column) is not supported yet.|
-| | | AUTO_INCREMENT is not supported yet. |
+| | | AUTO_INCREMENT is supported limitedly. The custom starting value is not supported yet.|
 |   | CREATE other projects | CREATE/DROP INDEX is not supported. |
 |   | ALTER | Not supported now.  |
 |   | DROP DATABASE | Same as MySQL. |
 |   | DROP TABLE | Same as MySQL. |
-| DML  | INSERT | LOW_PRIORITY, DELAYED, HIGH_PRIORITY are not supported now.  |
+||CREAT VIEW|The `with check option` clause is not supported yet.|
+| DML  | UPDATA| `UPDATE` is supported|
```

Comment:
`"UPDATE" is supported` is pointless. either remove this item, either put "Same as MySQL", either point out the incompatible parts with MySQL.